code/datums/mutantraces.dm

```diff
@@ -2219,7 +2219,9 @@ TYPEINFO(/datum/mutantrace/cow)
 	emote(var/act, var/voluntary)
 		switch(act)
 			if ("scream")
-				if (src.mob.emote_check(voluntary, 50) && !src.mob.bioHolder.HasEffect("mute"))
+				if (src.mob.bioHolder.HasEffect("mute"))
+					return // use muted scream emote handling
```

Comment:
is this really using base? it's not calling parent

Reply:
Mutantrace emote overrides work by calling `src.mutantrace.emote(act, voluntary)` which sets the `message` variable here:
https://github.com/goonstation/goonstation/blob/master/code/mob/living/carbon/human/procs/emote.dm#L39

If the return is empty, the `message` variable is null and thus we branch into the default carbon/human scream handling here:
https://github.com/goonstation/goonstation/blob/master/code/mob/living/carbon/human/procs/emote.dm#L48